src/main/java/io/jenkins/plugins/cascgroovy/GroovyScriptCaller.java

```diff
@@ -1,36 +1,28 @@
 package io.jenkins.plugins.cascgroovy;
 
-import jenkins.model.Jenkins;
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
-import hudson.EnvVars;
+import groovy.lang.Binding;
+import groovy.lang.GroovyShell;
 import hudson.Extension;
-import io.jenkins.plugins.casc.Attribute;
-import io.jenkins.plugins.casc.ConfigurationContext;
-import io.jenkins.plugins.casc.ConfiguratorException;
-import io.jenkins.plugins.casc.Configurator;
-import io.jenkins.plugins.casc.RootElementConfigurator;
+import io.jenkins.plugins.casc.*;
 import io.jenkins.plugins.casc.impl.attributes.MultivaluedAttribute;
 import io.jenkins.plugins.casc.model.CNode;
-import io.jenkins.plugins.casc.model.Sequence;
+import io.jenkins.plugins.casc.model.Mapping;
+import jenkins.model.Jenkins;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
-import groovy.lang.GroovyShell;
-import groovy.lang.Binding;
 
-import java.io.PrintWriter;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
+
+import static io.vavr.API.Try;
+import static io.vavr.API.unchecked;
 
 
 /**
  * @author <a href="mailto:tomasz.szandala@gmail.com">Tomasz Szandala</a>
  */
-@Extension(optional = true)
+@Extension(optional = true, ordinal = -50)
```

Comment:
The ordinal does not take affect until [JCasC v1.41](https://github.com/jenkinsci/configuration-as-code-plugin/releases/tag/configuration-as-code-1.41)

Reply:
Think it is better to load this after JobDSL.

WDYT @timja @oleg-nenashev @szandala ?

```suggestion
@Extension(optional = true, ordinal = -60) // Ordinal -60 Ensure it is loaded after JobDSL
```

Reply:
yeah makes sense